src/rules/selector-descendant-combinator-no-non-space/__tests__/index.js

```diff
@@ -0,0 +1,48 @@
+import {
+  messages,
+  ruleName,
+} from ".."
+import rules from "../../../rules"
+import { testRule } from "../../../testUtils"
+
+const rule = rules[ruleName]
+
+testRule(rule, {
+  ruleName,
+  config: [true],
+
+  accept: [ {
+    code: ".selector1.selector2 {}",
+  }, {
+    code: ".selector1 .selector2 {}",
+  }, {
+    code: ".selector1  .selector2 {}",
+  }, {
+    code: ".selector1>.selector2 {}",
+  }, {
+    code: ".selector1 > .selector2 {}",
+  }, {
+    code: ".selector1  >  .selector2 {}",
```

Comment:
Right you are.

It's the test three above, i.e. `.selector1  .selector2 {}`, that should be rejected.